data/docs_markdown/IO_Coin.md

```
@@ -0,0 +1,74 @@
+# IO Coin
+What is IO Coin ($IO)?
+IO Coin ($IO) is the native cryptocurrency of the IOG Network, powering its ecosystem and facilitating seamless interactions between key participants. These participants include:
+### [](https://docs.io.net/docs/io-coin#gpu-renters-users)
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)